identity/app/idapiclient/IdApiClient.scala

```diff
@@ -166,6 +166,8 @@ class IdApiClient(
   def deleteTelephone(auth: Auth): Future[Response[Unit]] =
     delete("user/me/telephoneNumber", Some(auth)) map extractUnit
 
+  def unsubscribeFromAllEmailsAndOptoutMarketingConsents(auth: Auth): Future[Response[Unit]] =
+    post("remove/consent/all", Some(auth)).map(extractUnit)
```

Comment:
I have a feeling that this endpoint required client token access and cookie access - so not sure this will work without adding the client token?

Reply:
It works with just user auth, tested it locally.

Reply:
huh, the API code appears to expect both - does this send a client access token as well maybe?

Reply:
Looks like `IdApiClient#buildHeaders` always adds the client auth token.